### ISSUE TITLE
Generic type 'ModuleWithProviders

### ISSUE THREAD

**Issue by kamleshdarteweb** [dominique-mueller/angular-notifier#188]
I am facing issue ERROR in node_modules/angular-notifier/lib/notifier.module.d.ts:32:51 - error TS2314: Generic type 'ModuleWithProviders<T>' requires 1 type argument(s).
    
    32     static withConfig(options?: NotifierOptions): ModuleWithProviders;
Please help to sort out
I am using latest angular

**Comment by erassynathingo** [comment]
Im also having this issue. Using angular 10.1.0

**Comment by kylecannon** [comment]
you can get around this issue by running ngcc as a postinstall command to compile everything to ivy.. but it's still not the best. 

@dominique-mueller can we get this updated to angular 10?

**Comment by LaysDragon** [comment]
> you can get around this issue by running ngcc as a postinstall command to compile everything to ivy.. but it's still not the best.
> 
> @dominique-mueller can we get this updated to angular 10?
I try whole day, but seems ngcc or other workaround can't handle it properly .
It still failed with follow log after I add ngcc into postinstall script...any idea? 
angular version:10.1.3
node version: 12.16.1
```
ERROR in ./src/app/app.module.ts
Module not found: Error: Can't resolve 'angular-notifier/lib/notifier.module' in 'D:\Coding\***\src\app'
```

**Comment by dominique-mueller** [comment]
Released with **[7.0.0](https://github.com/dominique-mueller/angular-notifier/releases/tag/7.0.0)**.
